includes/class-genesis-responsive-slider-admin.php

```diff
@@ -0,0 +1,432 @@
+						<select id="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[orderby]" name="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[orderby]">
+							<option style="padding-right:10px;" value="date" <?php selected( 'date', Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'orderby' ) ); ?>><?php esc_html_e( 'Date', 'genesis-responsive-slider' ); ?></option>
+							<option style="padding-right:10px;" value="title" <?php selected( 'title', Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'orderby' ) ); ?>><?php esc_html_e( 'Title', 'genesis-responsive-slider' ); ?></option>
+							<option style="padding-right:10px;" value="ID" <?php selected( 'ID', Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'orderby' ) ); ?>><?php esc_html_e( 'ID', 'genesis-responsive-slider' ); ?></option>
+							<option style="padding-right:10px;" value="rand" <?php selected( 'rand', Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'orderby' ) ); ?>><?php esc_html_e( 'Random', 'genesis-responsive-slider' ); ?></option>
+						</select>
+					</p>
+
+				</div>
+
+				<hr class="div" />
+
+				<h4><?php esc_html_e( 'Transition Settings', 'genesis-responsive-slider' ); ?></h4>
+
+					<p>
+						<label for="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_timer]"><?php esc_html_e( 'Time Between Slides (in milliseconds)', 'genesis-responsive-slider' ); ?>:
+						<input type="text" id="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_timer]" name="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_timer]" value="<?php echo esc_html( Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'slideshow_timer' ) ); ?>" size="5" /></label>
+					</p>
+
+					<p>
+						<label for="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_delay]"><?php esc_html_e( 'Slide Transition Speed (in milliseconds)', 'genesis-responsive-slider' ); ?>:
+						<input type="text" id="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_delay]" name="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_delay]" value="<?php echo esc_html( Genesis_Responsive_Slider::genesis_get_responsive_slider_option( 'slideshow_delay' ) ); ?>" size="5" /></label>
+					</p>
+
+					<p>
+						<label for="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_effect]"><?php esc_html_e( 'Slider Effect', 'genesis-responsive-slider' ); ?>:
+						<?php esc_html_e( 'Select one of the following:', 'genesis-responsive-slider' ); ?>
+						<select name="<?php echo esc_html( GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD ); ?>[slideshow_effect]" id="<?php echo esc_html( GENESIS_SETTINGS_FIELD ); ?>[slideshow_effect]">
```

Comment:
I believe `GENESIS_SETTINGS_FIELD` should be `GENESIS_RESPONSIVE_SLIDER_SETTINGS_FIELD`